src/components/CodeAndPreview.tsx

```diff
@@ -50,6 +53,13 @@ export function CodeAndPreview({ snippet }: Props) {
 
   const { toast } = useToast()
 
+  const importsMap = useMemo(
```

Comment:
variable transparency

```suggestion
  const userImports = useMemo(
```